spec/models/participant_outcome/npq_spec.rb

```diff
@@ -80,6 +80,60 @@
     end
   end
 
+  describe ".to_send_to_qualified_teachers_api" do
+    subject(:result) { described_class.to_send_to_qualified_teachers_api }
+
+    context "when a declaration is not closed" do
```

Comment:
worth noting outcomes only ever exist on completed NPQ declarations. NPQ completed declarations will never exist without an outcome. I propose we amend/remove this spec to avoid confusion

Reply:
discuss in person: yea no longer needed